website/data/api-docs-nav-data-hidden.json

```diff
@@ -1,20 +0,0 @@
-[
```

Comment:
The `*-nav-data-hidden.json` files were part of the temporary solution, so removing them. These nav-data items have been added to the proper `nav-data` file, with the `"hidden"` prop in place to reflect current state.